python/sample.py

```diff
@@ -0,0 +1,157 @@
+'''
+Sample python script for Scheduled Events.
+
+Continues to run GET and POST (if approved by user) request to Scheduled Events endpoint.
+Press Ctrl + C and then Enter to exit.
+'''
+# Copyright 2014 Microsoft Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import urllib.request
+import urllib.parse
+import sys
+import os
+import argparse
+import winreg as wreg
+
+def get_scheduled_events(address, headers):
+    '''
+    Make a GET request and return the response received.
+    '''
+    request = urllib.request.Request(address, headers=headers)
+    return urllib.request.urlopen(request)
+
+def post_scheduled_events(address, data, headers):
+    '''
+    Make a POST request.
+    '''
+    request = urllib.request.Request(address, data=str.encode(data), headers=headers)
+    urllib.request.urlopen(request)
+
+def make_address(ip_address):
+    '''
+    Returns the address for scheduled event endpoint from the IP address provided.
+    '''
+    return 'http://' + ip_address + '/metadata/scheduledevents?api-version=2017-03-01'
+
+def check_ip_address(address, headers):
```

Comment:
**Nit/Optional**: Some of this code obfuscates the usage of scheduled events with extra details for error handling and endpoint discovery. Might be nice to move some of these functions related to discovering the endpoint (like **check_ip_address** and **get_ip_address_reg_env**) to a separate module. This way the main use of scheduled events is highlighted more clearly. 

Totally up to you though :) overall looks good